lib/index.js

```diff
@@ -190,11 +177,35 @@ function determineTargetPath(filepath, { format, suffix = "" }) {
 	return path.join(directory, `${basename}${suffix}${extension}${format}`);
 }
 
+/**
+ * @param {...string} extensions
+ * @returns {Filter}
+ */
 function withFileExtension(...extensions) {
 	return filename => extensions.includes(extname(filename));
 }
 
-// extname follows this annoying idea that the dot belongs to the extension
+/**
+ * extname follows this annoying idea that the dot belongs to the extension
```

Comment:
As a ranty comment, this seemed fine - as a docstring, it seems more confusing than helpful? At least I had to spend a few cycles trying to decode the intent here.

However, I believe faucet-static does the same thing, so perhaps that should become a reusable utility (i.e. it shouldn't be inlined after all 🙈 )?